### DISCUSSION TITLE
How can I use `RANK()` which is in sql query?

### DISCUSSION THREAD

**Discussion by ogadra** [prisma/prisma#13002] [Q&A] [answered]
Hello

I want to get ranking from SQLite DB.
In RawSQL, I can get this in this code.

```SQLite3
SELECT
  user
  RANK() OVER(ORDER BY score DESC)
    AS 'rank' FROM Table
ORDER BY score DESC
```

But, I don't know get rank without using `prisma.$rawQuery`. Can I get this without `prisma.$rawQuery` or do I have to suggest an idea in Feature request?

**Comment by austincrim** [comment, marked as answer]
Hey @ogadra 👋,

We do not support `RANK()` yet without using a raw query. Feel free to open a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) and tell us more about your use case!

**Comment by ogadra** [comment]
I wrote.
https://github.com/prisma/prisma/issues/13024

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
